File: src/main/java/net/frozenblock/configurableeverything/biome_placement/util/BiomeSourceExtension.java
package net.frozenblock.configurableeverything.biome_placement.util;

import net.minecraft.core.Holder;
import net.minecraft.world.level.biome.Biome;
import java.util.List;

public interface BiomeSourceExtension {

	void updateBiomesList(List<Holder<Biome>> biomesToAdd, List<Holder<Biome>> biomesToRemove);
}

File: src/main/java/net/frozenblock/configurableeverything/biome_placement/util/BiomeSourceExtension.kt
package net.frozenblock.configurableeverything.biome_placement.util

import net.minecraft.core.Holder
import net.minecraft.world.level.biome.Biome

fun interface BiomeSourceExtension {
    fun updateBiomesList(biomesToAdd: List<Holder<Biome>>, biomesToRemove: List<Holder<Biome>>)
}
